shared/src/commonMain/kotlin/co/touchlab/kampstarter/Koin.kt

```diff
@@ -15,11 +16,11 @@ fun initKoin(appDeclaration: KoinAppDeclaration = {}) = startKoin {
 }
 
 private val coreModule = module {
-    single { DatabaseHelper(get(), getWith("DatabaseHelper")) }
+    single { DatabaseHelper(get(), getWith("DatabaseHelper"), Dispatchers.Main) }
```

Comment:
Should this be using `Dispatchers.Default` or some other background dispatcher instead of `Main`?

Reply:
Changed to Default